jaxrs-clients/src/main/java/com/palantir/remoting3/jaxrs/AbstractFeignJaxRsClientBuilder.java

```diff
@@ -88,17 +95,12 @@
                                                 new JacksonEncoder(objectMapper)))))
                 .decoder(createDecoder(objectMapper, cborObjectMapper))
                 .errorDecoder(FeignSerializableErrorErrorDecoder.INSTANCE)
-                .client(target.wrapClient(createOkHttpClient()))
-                .retryer(target)
+                .client(createOkHttpClient(config.uris()))
                 .options(createRequestOptions())
                 .logger(new Slf4jLogger(JaxRsClient.class))
                 .logLevel(Logger.Level.BASIC)
                 .requestInterceptor(UserAgentInterceptor.of(userAgent))
-                .target(target);
-    }
-
-    private <T> FailoverFeignTarget<T> createTarget(Class<T> serviceClass, List<String> uris) {
-        return new FailoverFeignTarget<>(uris, serviceClass, NeverRetryingBackoffStrategy.INSTANCE);
+                .target(serviceClass, primaryUri);
```

Comment:
I don't think so, it throughs `java.lang.UnsupportedOperationException: Request with non-absolute URL not supported with empty target` before hitting the okhttp code path that could inject a URL.